src/common/ContractInteractor.ts

```diff
@@ -350,6 +350,14 @@ export default class ContractInteractor {
     return await this._getPastEvents(relayHub.contract, names, extraTopics, options)
   }
 
+  async getTokensForRelayVerifier (): Promise<EventData[]> {
+    return await this._getPastEvents(this.relayVerifierInstance.contract, ['AllowedToken'], [], { fromBlock: 0 })
```

Comment:
It is not a good idea from the scalability point of view to read events from block 0.